tensorflow/lite/micro/kernels/reduce.cc

```diff
@@ -29,6 +29,22 @@ namespace reduce {
 
 constexpr int kMaxNumberOfAxis = 4;
 constexpr int kMaxNumberOfReducedAxis = 2;
+enum ReduceType {
+  kMax,
+};
```

Comment:
I would suggest removing this (and the correponding switch statements, template params ...) Its a lot of complexity that is not necessary at this time.

Reply:
done